javascript-data-model/order-history.js

```diff
@@ -0,0 +1,88 @@
+
+var orderHistory = ([
+  {
+    orderPlaced: 'Auguest 4, 2020',
+    total: 34.00,
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-87722232',
+    deliveredDate: 'Delievered Aug,8 2020',
+    deliveryMethod: ' ',
+    item: {
+      description1: ['Javascript for Impatient programmers'],
+      author: ['Rauschmayer Dr Axel'],
+      rtrnWdindow: ['Sep 7,2020'],
+      price: [31.00],
+      buyItAgain: [true],
+      viewItem: [true],
+      productReview: [true],
+      orderRetails: [true],
+      invocie: [true],
+      productSupport: [false]
+    }
+  },
+
+  {
+    orderPlaced: 'July 19,2020',
+    total: 44.53,
+    shipTo: 'JS Masher',
+    orderNum: '133-9984268-1280257',
+    deliveredDate: 'Delievered Jul 20, 2020',
+    deliveryMethod: 'Your package was delivered It was handed directly handed to a resident',
+    item: {
+      description: ['The Timeless Way of Building'],
+      author: ['Alexander Christopher'],
+      rtrnWindow: ['Aug 19,2020'],
```

Comment:
The values of your item properties don't need to be in arrays, but each item that is in the order should be in an array, as described in my comment above.